### ISSUE TITLE
Mixins are not applied

### ISSUE THREAD

**Issue by patric-eberle** [vue-styleguidist/vue-styleguidist#80]
As described [here](https://github.com/vue-styleguidist/vue-styleguidist/blob/master/docs/Configuration.md#mixins) I defined a mixin for the Styleguidist. But it seems like the object is not mixed with the component instance. You can reproduce the issue if you try to add a mixin on the styleguide.config.js or the imported mixin in your [demo repository](https://github.com/rafaesc/vue-styleguidist-demo).

Code inside the imported mixin file seems to be executed though.

```javascript
module.exports = {
  components: 'src/**/example.vue',
  mixins: [
    'src/mixins.js', // <-- the data properties from this file aren't mixed in neither
    {
      created() { // <-- This is never called
        console.info('component created')
      }
    }
  ],
  webpackConfig: {
    module: {
      loaders,
    }
  },
  serverPort: 6062
};
```

**Comment by Infeligo** [comment]
This problem first occurred in 1.4.0.

**Comment by rafaesc** [comment]
Thanks @patric-eberle, @Infeligo 

I fixed it, on the version 1.4.7

Regards!
